in-market-test/helper-nodes/setup_instructions.md

```diff
@@ -1,44 +1,83 @@
-Here are the instructions for setting up node for running IPA. 
+Here are the instructions for setting up node for running IPA.
 
 ## Step 1: Download code and run a sample test
 1. If you do not have Rust installed on your machine, you can follow [these](https://doc.rust-lang.org/book/ch01-01-installation.html#installation) instructions to do so.
 2. Download the code from https://github.com/private-attribution/ipa
-   You can do this by running the command on the terminal: 
+   You can do this by running the command on the terminal:
    `git clone https://github.com/private-attribution/ipa`
 3. This will create a folder named “ipa”. Go inside the folder.
-   
+
    Command: `cd ipa`
-4. Build the Rust code 
-   
+4. Build the Rust code
+
    Command: `cargo build`
-5. Running a sample test : 
-   
+5. Running a sample test :
+
    Command: `cargo bench --bench oneshot_ipa --features="enable-benches" --no-default-features`
-   
+
 ## Step 2: Setup helper configuration
 1. Provide details of how the machine can be accessed from public internet. For this, we would need to know
    i.  IP address/DNS which is accessible from public internet (i.e. firewall allows connecting to the helper port (443))
-   ii. Confirm if machine is able to listen on port 443 
+   ii. Confirm if machine is able to listen on port 443
+
+2. Installing/Upgrading Docker
+   - If you need to install docker, follow the instructions [here](https://docs.docker.com/engine/install/).
+   - If you already have docker, ensure that you are running at least Docker Version 20.10.22. You can check version by running command `docker -v` on the terminal.
+
+3. Download docker image of IPA from shared folder. It will be uploaded in an executable docker image which will have IPA code.
+
+   Link: TBD.
+
+   Alternatively, you can build the docker image directly from the Rust code with this command
+
+       `cd <path_to_ipa_source_code>`
+
+       `scripts/helper-image.sh --identity 1 --hostname localhost`
```

Comment:
this command generates an image for H1, if H2 and H3 want to do it, they need to either use `--identity 2|3` argument or regenerate the keys.

also `localhost` hostname is a setup suitable for local execution, for real helper parties, we need the DNS name here